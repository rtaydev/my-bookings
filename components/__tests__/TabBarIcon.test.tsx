import React from "react";
import renderer from "react-test-renderer";
import { TabBarIcon } from "../navigation/TabBarIcon";

it("renders correctly", () => {
  const tree = renderer
    .create(<TabBarIcon name="home" color="#000" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("applies correct icon name and color", () => {
  const tree = renderer
    .create(<TabBarIcon name="home" color="#000" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
