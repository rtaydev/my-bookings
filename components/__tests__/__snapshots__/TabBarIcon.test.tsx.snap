// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`applies correct icon name and color 1`] = `
<Text
  allowFontScaling={false}
  selectable={false}
  style={
    [
      {
        "color": "#000",
        "fontSize": 28,
      },
      [
        {
          "marginBottom": -3,
        },
        undefined,
      ],
      {
        "fontFamily": "ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      {},
    ]
  }
>
  î²ƒ
</Text>
`;

exports[`renders correctly 1`] = `<Text />`;
